src/WSRequestHandler_SceneCollections.cpp

```diff
@@ -12,26 +17,32 @@
  * @category scene collections
  * @since 4.0.0
  */
-RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest& request) {
-	if (!request.hasField("sc-name")) {
+RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest &request)
```

Comment:
References and pointers must have their qualifier aligned with the type:
```suggestion
RpcResponse WSRequestHandler::SetCurrentSceneCollection(const RpcRequest& request)
```